orchestrator-salt/src/main/resources/salt-common/salt/monitoring/scripts/logging-agent.sh.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+if [[ -f "/var/run/cdp-logging-agent/cdp-logging-agent.pid" ]]; then
```

Comment:
can't we decide this from salt? instead of checking at every run

Reply:
the process can stop later, from salt it would be only decided during the script generation, and this one also probably easier, more cheap then use a salt call for checking a service (from the bash script)